bpm/bonita-api/bonita-common-api/src/main/java/org/bonitasoft/engine/bpm/bar/actorMapping/BEntryAdapter.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (C) 2015 BonitaSoft S.A.
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ ******************************************************************************/
+
+package org.bonitasoft.engine.bpm.bar.actorMapping;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.bonitasoft.engine.bar.BEntry;
+
+/**
+ * Created by mazourd on 15/07/15.
+ */
+public class BEntryAdapter extends XmlAdapter<BEntryAdapter.Entry, BEntry<String, String>> {
```

Comment:
BEntry was already there before the PR (not at the same place but still). And it is used by quite a lot of classes throughout the engine. Though I agree it needs to be refactored, it will go far beyond what this PR (already pretty big in itself) is about.